### PR TITLE
Fix Divide by 0 error

### PR DESCRIPTION
**Pull request by brymut**
Added a greater than check to totalDeposit value in liquidity Rewards APY calculation as its currently returning a 0 BigNumber on mainnet not sepolia though, causing a divide by 0 error. 

## Related GitHub Issue

https://github.com/nation3/citizen-app/issues/295

## How Has This Been Tested?

- [x] Status checks pass (lint, build, test)
- [ ] Works on Sepolia preview deployment
- [ ] Works on Mainnet preview deployment

### CODE REVIEW

**Comment by brymut** [comment]
Working on mainnet deployment: https://citizen-app-mainnet-2kxkcgu7k-nation3dao.vercel.app/liquidity

**Review by aahna-ashina — CHANGES_REQUESTED** [review]
@brymut I'm getting an error at https://citizen-app-mainnet-gxcp27a7m-nation3dao.vercel.app/liquidity

```javascript
TypeError: Cannot read properties of undefined (reading 'gt')
```

**Comment by brymut** [comment]
> @brymut I'm getting an error at https://citizen-app-mainnet-gxcp27a7m-nation3dao.vercel.app/liquidity
> 
> ```js
> TypeError: Cannot read properties of undefined (reading 'gt')
> ```

Unable to reproduce, page using the link you've provided loads up fine for me.  Maybe it comes up when the value of `` is still undefined when the hook runs? @aahna-ashina 

Screenshot 2024-08-22 at 11 56 06

**Comment by aahna-ashina** [comment]
@brymut Please don't force-push changes: https://github.com/nation3/citizen-app/compare/a155dd663f3a90fc5419217c99986ccb3af9592f..517ec2733b60e3b48023602f4de600fc8312b199

It would be better if you fetch changes from the `main` instead of overwriting them.

File: ui/lib/liquidity-rewards.ts [hunk 1]
```diff
@@ -68,12 +68,14 @@ export function useLiquidityRewards({ nationPrice, poolValue, address }: any) {
 
   useEffect(() => {
     if (totalRewards && poolValue && totalDeposit && lpTokensSupply) {
-      setLiquidityRewardsAPY(
-        totalRewards
-          .mul(transformNumber(12 / months, NumberType.bignumber))
-          .mul(transformNumber(nationPrice, NumberType.bignumber, 2))
-          .div(poolValue.mul(totalDeposit).div(lpTokensSupply))
-      )
+      if (totalDeposit.gt(transformNumber(0, NumberType.bignumber))){
```
Comment (aahna-ashina) [inline ui/lib/liquidity-rewards.ts:71]: @brymut `totalDeposit > 0` does not work here?

**Review by aahna-ashina — APPROVED** [review]

File: ui/lib/liquidity-rewards.ts (hunk 1 above)
Comment (brymut) [reply ui/lib/liquidity-rewards.ts:71]: Since this is a big Number type, thought this would be more appropriate

**Comment by brymut** [comment]
> @brymut Please don't force-push changes: https://github.com/nation3/citizen-app/compare/a155dd663f3a90fc5419217c99986ccb3af9592f..517ec2733b60e3b48023602f4de600fc8312b199
> 
> It would be better if you fetch changes from the `main` branch instead of overwriting them.

Apologies @aahna-ashina, was trying to keep the changes to one commit and this is how I normally do it. Noted.

**Review by aahna-ashina — APPROVED** [review]

**Pull request merged**